fchost/fchost.go

```diff
@@ -57,6 +63,25 @@ func (fh *FilecoinHost) Bootstrap() error {
 	return nil
 }
 
+func (fc *FilecoinHost) Ping(ctx context.Context, pid peer.ID) bool {
+	r := <-fc.ping.Ping(ctx, pid)
+	return r.Error == nil
+}
+
+func (fc *FilecoinHost) GetAgentVersion(pid peer.ID) string {
```

Comment:
Abstracted AgentVersion fetching info. 
In case we're interested in an extra thing, it could receive the key as param and be generic.